components/results_champ.py

```diff
@@ -0,0 +1,9 @@
+import streamlit as st
+
+from components.results import compute_results
+from dtower.tourney_results.constants import Graph, Options, champ
+
+if __name__ == "__main__":
```

Comment:
Do let me know if this is used still. I'm looking at pages.py and all i see is calling compute_results() with different arguments, so hopefull we can remove results_{league}.py files